spec/haproxy/templates/haproxy_config/frontend_http_spec.rb

```diff
@@ -42,6 +42,30 @@
         expect(frontend_http).to include('bind :80 accept-proxy')
       end
     end
+
+    context 'when ha_proxy.expect_proxy_cidrs is not empty/nil and ha_proxy.accept_proxy is false' do
+      let(:properties) do
+        { 'accept_proxy' => false,
+          'expect_proxy_cidrs' => ['127.0.0.1/8'] }
```

Comment:
I would add an additional spec test to check that if the expect_proxy_cidrs is not explicitly set, then the line `tcp-request connection expect-proxy layer4 if { src -f /var/vcap/jobs/haproxy/config/expect_proxy_cidrs.txt` is not here. To prevent regressions in the future.

Reply:
@b1tamara Thanks Tamara, feel free to take a look again.